steps/steps.py

```diff
@@ -0,0 +1,96 @@
+    def check_current_value(context):
+        first_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number1"]'
+        )
+        second_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number2"]'
+        )
+
+        if first_number_input.get_attribute("value") != '':
+            first_number_input.clear()
+            if second_number_input.get_attribute("value") != "":
+                second_number_input.clear()
+        assert (
+            first_number_input.get_attribute("value") and
+            second_number_input.get_attribute("value")
+            ) == ''
+
+    use_step_matcher("re")
+
+    @step(
+        'you enter \'(?P<number>.*)\' as the (?P<field>first|second) '
+        'number in the calculator'
+        )
+    def enter_number(context, number, field):
+        if field == 'first':
+            first_number_input = context.browser.find_element_by_xpath(
+                '//input[@id="number1"]'
+            )
+            first_number_input.send_keys(number)
+            context.browser.first_number = int(number)
+        else:
+            second_number_input = context.browser.find_element_by_xpath(
+                '//input[@id="number2"]'
+            )
+            second_number_input.send_keys(number)
+            context.browser.second_number = int(number)
+    use_step_matcher('parse')
+
+    @step('you click the \'{operation}\' button')
+    def click_option(context, operation):
+        option = context.browser.find_element_by_xpath(
+            '//option[contains(., "{operation}")]'.format(operation=operation)
+        )
+        option.click()
+        context.browser.option = operation
+
+    @step('you click the equals button')
+    def click_equals(context):
+        equals = context.browser.find_element_by_xpath(
+            '//input[@id="calculate"]'
+        )
+        equals.click()
+
+    @step(
+        'the value displayed is the correct result for the two numbers entered'
+        )
+    def correct_answer(context):
+        answer = context.browser.find_element_by_xpath('//span[@id="answer"]')
+        answer_text = answer.text
+        print(answer_text)
+        if context.browser.option == 'plus':
+            assert answer_text == str(
+                context.browser.first_number + context.browser.second_number
+            )
+        elif context.browser.option == 'minus':
+            assert answer_text == str(
+                context.browser.first_number - context.browser.second_number
+            )
+        elif context.browser.option == 'times':
+            assert answer_text == str(
+                context.browser.first_number * context.browser.second_number
+            )
+        elif context.browser.option == 'divide':
+            if (context.browser.second_number != 0):
+                assert answer_text in str(
+                    context.browser.first_number /
```

Comment:
Uwaga na pułapkę niekompatybilności pythona2 i pythona3 przy znaku dzielenia! :)
To co napisałaś zadziała tylko w py3, gdzie `/` daje rzeczywisty wynik dzielenia. `3 / 2 == 1.5`. W pythonie2 `/` to "całkowite" dzielenie więc  tam `3 / 2 == 1`. W pythonie3 całkowite dzielenie to `//`. Dlatego ważne jest, by wiedzieć, na jakiego pythona się pisze :). Jeśli chcesz, by kod działał tak samo w py2 i w py3, to na początku pliku musiałabyś zaimportować `from __future__ import division` i wtedy `3 / 2` w obu pythonach będzie miało wartość `1.5`, a `3 // 2` będzie i tu i tu `1`. :)